manual/winamp/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,20 @@
+$ErrorActionPreference = 'Stop'
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+
+$pp = Get-PackageParameters
+$file = if ($pp.Light) { gi $toolsPath\*lite* } else { gi $toolsPath\*full* }
+
+$packageArgs = @{
+  packageName    = 'winamp'
+  fileType       = 'exe'
+  file           = $file
+  silentArgs     = '/S'
+  validExitCodes = @(0)
+  softwareName   = 'Winamp'
+}
+Install-ChocolateyInstallPackage @packageArgs
+ls $toolsPath\*.exe | % { rm $_ -ea 0; if (Test-Path $_) { sc "$_.ignore" }}
+
+$ini_path = "$Env:APPDATA\Winamp\winamp.ini"
+(gc $ini_path -Encoding UTF8) -replace '^NeedReg=1', 'NeedReg=0' | sc $ini_path -Encoding UTF8
```

Comment:
just curious, what is the purpose of this line?

Reply:
There is a startup screen after winamp is installed. You select skin on it and some other stuff. Nothing you can't do within winamp itself. If I install audio player I don't want it to do a splash when I run mp3, even the first time.

Reply:
Sounds good

Reply:
At the end it also tries to send some stats to winamp site which no longer functions

Reply:
And that is something I would have liked to disable even if it still worked.
But that may just be me.

Reply:
Yeah, it sux lama's ass :)